google/cloud/spanner/database_admin_client.h

```diff
@@ -39,6 +39,9 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 /**
  * Performs database administration operations on Spanner.
  *
+ * @deprecated Please use google::cloud::spanner::admin::InstanceAdminClient
```

Comment:
s/Instance/Database/

Reply:
Fixed here and elsewhere, also used `#` to turn them into links